commands/delete.py

```diff
@@ -48,14 +24,31 @@ def run(self, paths=None, *args, **kwargs):
             paths_to_display = [
                 [confirm_title, confirm_subtitle],
                 [cancel_title, cancel_subtitle],
-            ]
-            for path in self.paths:
-                paths_to_display.append([os.path.basename(path), path])
+            ] + [[os.path.basename(path), path] for path in self.paths]
```

Comment:
I agree that this is neat, but I'm pretty sure it's not as memory efficient as the previous version, because it creates a second list that is just used to be merged in the first one.

Although it shouldn't matter too much in this package's case because the lists typically won't be massive, I would prefer this:

```python
paths_to_display = [confirm, cancel]
paths_to_display.extend([os.path.basename(path), path] for path in self.paths)
```

Notice how there isn't any `[]` around the argument to extend, which means that it's going to create a generator instead of a list, ie. don't store the entire arguments, but as little as possible, probably one item at a time (i say probably because I don't actually know how it's really implemented, but that's idea with generators).